### PR TITLE
patch(bot-runners): Change default configs and patch gckms provider issue

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**
`web3.eth.net.getId()` does not return correctly for `_gkcms` provider unless its the second Promise returned in the Promise.all()'s in the bot runners.

**Summary**

Briefly summarize what changes were made to accomplish the motivation above.

**Details**

This may be unnecessary for some PRs. Catch-all for detailed explanations about the implementation decisions and implications of the change.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [X]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #XXXX

### CODE REVIEW

File: packages/funding-rate-proposer/index.js [hunk 1]
```diff
@@ -130,13 +130,13 @@ async function Poll(callback) {
...
-      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5,
+      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 3,
       // Default to 10 seconds in between error re-tries.
-      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
+      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 1,
       // Common price feed configuration passed along to all those constructed by proposer.
       commonPriceFeedConfig: process.env.COMMON_PRICE_FEED_CONFIG
         ? JSON.parse(process.env.COMMON_PRICE_FEED_CONFIG)
-        : {},
+        : null,
```
Comment (nicholaspai) [inline packages/funding-rate-proposer/index.js:139]: Changed to match other bot runners whch deafult their pricefeed configs to null

File: packages/optimistic-oracle/index.js [hunk 2]
```diff
@@ -126,13 +126,13 @@ async function Poll(callback) {
...
       // Default to 3 re-tries on error within the execution loop.
-      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5,
+      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 3,
       // Default to 10 seconds in between error re-tries.
-      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
+      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 1,
       // Common price feed configuration passed along to all those constructed by proposer.
       commonPriceFeedConfig: process.env.COMMON_PRICE_FEED_CONFIG
         ? JSON.parse(process.env.COMMON_PRICE_FEED_CONFIG)
-        : { lookback: 7200 },
```
Comment (nicholaspai) [inline packages/optimistic-oracle/index.js:135]: No reason to set default config with any params

File: packages/optimistic-oracle/index.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ async function run({
   optimisticOracleProposerConfig
 }) {
   try {
-    const [networkId, accounts] = await Promise.all([web3.eth.net.getId(), web3.eth.getAccounts()]);
+    const [accounts, networkId] = await Promise.all([web3.eth.getAccounts(), web3.eth.net.getId()]);
```
Comment (chrismaree) [inline packages/optimistic-oracle/index.js:35]: I hate this but glad it works.

**Review by chrismaree — APPROVED** [review]

File: packages/optimistic-oracle/index.js [hunk 4]
```diff
@@ -126,13 +126,13 @@ async function Poll(callback) {
...
-      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5,
+      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 3,
       // Default to 10 seconds in between error re-tries.
-      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
+      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 1,
       // Common price feed configuration passed along to all those constructed by proposer.
       commonPriceFeedConfig: process.env.COMMON_PRICE_FEED_CONFIG
         ? JSON.parse(process.env.COMMON_PRICE_FEED_CONFIG)
-        : { lookback: 7200 },
+        : null,
```
Comment (mrice32) [inline packages/optimistic-oracle/index.js:135]: Why null rather than empty object?

File: packages/optimistic-oracle/index.js [hunk 5]
```diff
@@ -126,13 +126,13 @@ async function Poll(callback) {
       // Default to 1 minute delay. If set to 0 in env variables then the script will exit after full execution.
       pollingDelay: process.env.POLLING_DELAY ? Number(process.env.POLLING_DELAY) : 60,
       // Default to 3 re-tries on error within the execution loop.
-      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5,
+      errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 3,
       // Default to 10 seconds in between error re-tries.
-      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
+      errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 1,
```
Comment (mrice32) [inline packages/optimistic-oracle/index.js:131]: Why not 2 or 3? 10 seems high, but 1 seems a little low. What do we do in the other bots?

File: packages/funding-rate-proposer/index.js (hunk 1 above)
Comment (mrice32) [inline packages/funding-rate-proposer/index.js:139]: Same questions here.

**Review by mrice32 — APPROVED** [review]
Looks good!

File: packages/optimistic-oracle/index.js (hunk 4 above)
Comment (nicholaspai) [reply packages/optimistic-oracle/index.js:135]: Just to be consistent with other bots, it doesn't really matter and I can just remove to reduce diff?

File: packages/optimistic-oracle/index.js (hunk 5 above)
Comment (chrismaree) [reply packages/optimistic-oracle/index.js:131]: this is the same as the other bots.

Comment (nicholaspai) [reply packages/optimistic-oracle/index.js:131]: we do 1 hence why i changed to this

File: packages/optimistic-oracle/index.js (hunk 4 above)
Comment (chrismaree) [reply packages/optimistic-oracle/index.js:135]: I think the main reasoning is downstream the bots check against null rather than empty object.

**Pull request merged**